### ISSUE TITLE
Commands without `summary` does not even show command name

### ISSUE THREAD

**Issue by judofyr** [ddfreyne/cri#92]
A simple command like this:

```ruby
require 'cri'

c = Cri::Command.define do
  name 'test'
  #summary 'foo'

  run do |opts, args, cmd|
    puts cmd.help
  end
end

c.run(ARGV)
```

Will not output anything:

```
$ ruby wat.rb

```

Once I uncomment the `summary` call, then it starts showing. I understand that summaries are useful for the end-user, but it would be nice if it at least showed the command name (instead of a blank line).

**Comment by ddfreyne** [comment]
Good catch — agreed that it should print at least something.

**Comment by ddfreyne** [comment]
Here’s a fix: #93. @judofyr Can you take a look and see whether this fits your needs?
